yarn-project/simulator/src/avm/journal/journal.ts

```diff
@@ -33,12 +34,12 @@ export class AvmPersistableStateManager {
     /** Reference to node storage */
     private readonly worldStateDB: WorldStateDB,
     /** Side effect trace */
-    private readonly trace: PublicSideEffectTraceInterface,
-    /** Public storage, including cached writes */
     // TODO(5818): make private once no longer accessed in executor
-    public readonly publicStorage: PublicStorage,
+    public readonly trace: PublicSideEffectTraceInterface,
```

Comment:
Made public for now.... Otherwise I need to keep adding passthrough functions to the trace. Eventually, the the only place outside the AVM where the trace will be accessed will be in the enqueued calls processor where it is first constructed. At that point we should be able to make this private again.